content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Istio Custom Certificate Authority Integration using Kubernetes CSR API [experimental]
```

Comment:
Title is very long. How about shortening:

```suggestion
title: Custom CA Integration using Kubernetes CSR [experimental]
```

Reply:
done

Reply:
You'll need to run `make gen` again, because of the doc title change